File: tests/fixtures/swift/input.swift
import Foundation

protocol Drawable {
    func draw() -> String
}

class Shape: Drawable {
    var area: Double = 0

    init(area: Double) {
        self.area = area
    }

    func draw() -> String {
        return "shape"
    }

    static func create() -> Shape {
        return Shape(area: 0)
    }
}

struct Point {
    var x: Int = 0
}

enum Color {
    case red
    case green
}

func greet(name: String) -> String {
    return "hi"
}

File: tests/fixtures/rust/input.rs
pub struct Point {
    pub x: i32,
    y: i32,
}

pub enum Color {
    Red,
    Green,
}

pub trait Drawable {
    fn draw(&self) -> String;
}

impl Point {
    pub fn new(x: i32) -> Self {
        Point { x, y: 0 }
    }
}

pub fn distance(a: i32, b: i32) -> i32 {
    a - b
}

mod util {}
